docs/reference/app-parameters.md

```diff
@@ -44,6 +44,10 @@ Sets the routing algorithm used for the services within the application.  Defaul
 
 Specifies the HTTP codes that healthy targets must use when responding to an HTTP health check. You can specify values between 200 and 499, and the default value is `200-399,401`. You can specify multiple values (for example, `200,202`) or a range of values (for example, `200-299`).
 
+### LoadBalancerGrpcSuccessCodes
+
+Specifies the GRPC codes that healthy targets must use when responding to an GRPC health check.  You can specify values between 200 and 499, and the default value is \"12\". You can specify multiple values (for example, \"12,13\") or a range of values (for example, \"10-99\").
```

Comment:
If the value range is between 200-499, then how can you specify 12,13, etc? 😆

Reply:
yeah sorry about that
will push the fix